document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -329,14 +366,15 @@ private StringBuilder prepareFieldAccessorExpr(String fieldName) {
   }
 
   @VisibleForTesting
-  protected String parseCompositeFilter(Filter filter, Params.Builder paramsBuilder) {
+  protected String parseCompositeFilter(
```

Comment:
Can you please add the `@Nullable` notation for `aliasPrefix`?

Reply:
Done